### PR TITLE
feat: Added run name for pdm lock action to easily view it

### PR DESCRIPTION
**Pull request by chandrasekharan-zipstack**
## What

- Added run name to easily view the service being worked with

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No

## Notes on Testing

- Checked by triggering manually

## Screenshots
Before
image

After
image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by kirtimanmishrazipstack — APPROVED** [review]
LGTM

**Review by Deepak-Kesavan — APPROVED** [review]

File: .github/workflows/pdm-lock-automation.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ on:
 
 jobs:
   check_pyproject_changes:
+    name: ${{ matrix.directory }} PDM lock update
```
Comment (ritwik-g) [inline .github/workflows/pdm-lock-automation.yaml:24]: @chandrasekharan-zipstack is this the correct place to add this? How have we tested this?

Comment (ritwik-g) [reply .github/workflows/pdm-lock-automation.yaml:24]: Will a manual trigger help to check this?

Comment (chandrasekharan-zipstack) [reply .github/workflows/pdm-lock-automation.yaml:24]: Triggered manually to check this and also updated the PR description @ritwik-g 
https://github.com/Zipstack/unstract/actions/runs/10611169561

**Review by ritwik-g — APPROVED** [review]

**Pull request merged**